src/authentication/AuthenticationSection.tsx

```diff
@@ -163,9 +168,22 @@ export const AuthenticationSection = () => {
     </>
   );
 
-  const AliasRenderer = ({ id, alias, builtIn }: AuthenticationType) => (
+  const AliasRenderer = ({
+    id,
+    alias,
+    usedBy,
+    builtIn,
+  }: AuthenticationType) => (
     <>
-      <Link to={`${url}/${id}`} key={`link-{id}`}>
+      <Link
+        to={toFlow({
+          realm,
+          id: id!,
+          usedBy: usedBy.type || "notInUse",
+          builtIn: builtIn ? "builtIn" : undefined,
+        })}
+        key={`link-{id}`}
```

Comment:
This template string is missing an expression.

```suggestion
        key={`link-${id}`}
```